here_for_change/municipalities/templates/municipalities/ward_detail.html

```diff
@@ -95,11 +95,11 @@ <h2 class="text-2xl md:text-3xl merriweather font-bold text-white text-center">
               <img class="rounded-lg w-20 h-20" src="https://assets.website-files.com/61a46f92a4845f1ed45ec01d/61d7e311bea0c7b4d41234e6_danny-europa.png"/>
               <div class="grow">
                 <div class="flex items-center justify-between">
-                  <p class="font-bold">Danny Europa</p>
+                  <p class="font-bold">{{content.councillor.name.value}}</p>
```

Comment:
@Hazmid see how this pulls from the "dummy" data setup in the view below.

Reply:
Fascinating. I noticed they were no imports. I like how clean Django is.